content/en/faqitems/project-1-bottlerocket-and-firecracker.markdown

```diff
@@ -0,0 +1,16 @@
++++
+question = "What's the difference between Firecracker and Bottlerocket?"
+group = "Project"
++++
+
+Quite a lot!
+[Firecracker](https://firecracker-microvm.github.io/) is a *virtualization* technology and Bottlerocket is an *operating system*. From the [Firecracker FAQ](https://firecracker-microvm.github.io/#faq):
+
+> Firecracker is an alternative to QEMU that is purpose-built for running serverless functions and containers safely and efficiently, and nothing more.
+
+Bottlerocket does not uses Firecracker.
```

Comment:
nit: "uses"

There's an open issue for Firecracker support so it's not a foregone conclusion that this will always be true, especially as out-of-tree builds make the distro easier to customize and experiment with.

For me this is like saying "Bottlerocket does not use Docker" - it's true except when it's not.